### ISSUE TITLE
Send / Export issue with Nuclear Medicine Images

### ISSUE THREAD

**Issue by jpm49** [horosproject/horos#591]
When I select a nuclear medicine exam for sending or exporting, only the first image of the tomographic series (acquired and reconstructed, gated or not gated) and of the dynamic series are transmitted. CT series are correctly transmistted with all the images. Static 2D images of NM and secondary capture are also well transmitted...

**Comment by orr721** [comment]
That is strange. I work with NM images routinely and have never experienced something like this (Horos v4.0.0 RC4, MacOS 10.13.6) - dynamic, static, WB, recon & raw SPECT, etc. all coming from GE scanners. I suspect the best would be to provide some anonymized sample data for developers to have a look on it.

**Comment by jensen0914** [comment]
Yes, a sample exam that demonstrates the problem would be useful.

**Comment by jpm49** [comment]
Finally, i think there is no issue (sorry for that). 
There is only a difference in the way where images are saved in the different modalities. 
For tomographic series in PT or CT, there is one .dcm file for one slice in the series. In NM, for tomographic acquisition or reconstruction series or for dynamic series, there is only one .dcm file for all the slices of the series.
As an illustration, I send attached one single .dcm file wich correspond to an entire tomoscintigraphic reconstruction and two screen captures of an Horos send with 1094 images in the study and "only" 528 files to send. 
Again sorry, but I was not aware of this difference of comportement between modalities.
Have a nice day.
 
<img width="621" alt="Capture d’écran 2020-09-21 à 11 42 12" src="https://user-images.githubusercontent.com/71459515/93754888-b250df00-fc02-11ea-8fe8-2ad7ee6a7264.png">
<img width="535" alt="Capture d’écran 2020-09-21 à 11 42 31" src="https://user-images.githubusercontent.com/71459515/93754916-be3ca100-fc02-11ea-848f-211286095536.png">
[IM-0003-0081.dcm.zip](https://github.com/horosproject/horos/files/5254711/IM-0003-0081.dcm.zip)

**Comment by jpm49** [comment]
NB : it's the same for tomoscintigraphic gated series. For exemple, a myocardial study with 2557 images has only 8 .dcm files to send.
